app/models/repository.rb

```diff
@@ -336,7 +336,7 @@ def check_status
       projects.each do |project|
         next unless %w[bower go elm alcatraz julia nimble].include?(project.platform.downcase)
 
-        project.update_attribute(status: "Removed", audit_comment: "Response 404")
+        project.update(status: "Removed", audit_comment: "Response 404")
```

Comment:
if a Project gets saved as "Removed", we want callbacks to run so that `ProjectUpdatedWorker` runs